week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,60 @@
+// PART-1
+
+// Clicking on the buttons should change the "theme" of the website
+
+// Blue Button
+
+let btnBlue = document.querySelector("#blueBtn");
+btnBlue.addEventListener("click", ChangeColor);
+function ChangeColor() {
+  document.querySelector(".jumbotron").style.background = "#588FBD";
```

Comment:
This works well but it could be a bit more readable. How about storing all the elements in variables that you can then reuse.
That way, you won't have to write the same code over and over again.
eg 
```
let jumbotron = document.querySelector(".jumbotron");
jumbotron.style.background="white";
```
This will keep you code cleaner  😉